backend/fms_core/services/library.py

```diff
@@ -48,3 +51,96 @@ def create_library(library_type, index, platform, strandedness, library_size=Non
         errors.append(';'.join(e.messages))
 
     return library, errors, warnings
+
+
+def convert_library(process, platform, sample_source, concentration, library_size, container_destination,
+                    coordinates_destination, volume_used, volume_destination, execution_date, comment):
+    library_destination = None
+    errors = []
+    warnings = []
+
+    if not process:
+        errors.append(f"Process is required.")
+    if not platform:
+        errors.append(f"Platform is required.")
+    if not library_size:
```

Comment:
I don't think there is a reason to force a library size here at the service level.